docs/root/operations/admin.rst

```diff
@@ -475,6 +475,26 @@ modify different aspects of the server:
   Buckets do not include values from other buckets with smaller upper bounds;
   the previous bucket's upper bound acts as a lower bound. Compatible with ``usedonly`` and ``filter``.
 
+  .. http:get:: /stats?histogram_buckets=detailed
+
+  Shows histograms as both percentile summary data, and raw bucket
+  data, up to 64 buckets. If the raw bucket data has more than 64
+  buckets, then adjacent buckets are combined until 64 is reached.
+
+  Example output
+  .. code-block:: text
+
+    http.admin.downstream_rq_time:
+      totals=1,0.25:25, 2,0.25:9
+      intervals=1,0.25:2, 2,0.25:3
+      summary=P0(1,1) P25(1.0625,1.034) P50(2.0166,1.068) P75(2.058,2.005) P90(2.083,2.06) P95(2.091,2.08) P99(2.09,2.09) P99.5(2.099,2.098) P99.9(2.099,2.099) P100(2.1,2.1)
+
+  Each bucket is shown as `min_value,width:count`. In the above example there are two
```

Comment:
I'd slightly prefer referring to it as `lower bound` instead of `min value`, but I may be splitting hairs

Reply:
I'll change that throughout; very stl-ish.